packages/cli/src/commands/set-admin.ts

```diff
@@ -7,29 +8,48 @@ import ConfigVariablesInitializer from '../models/initializer/ConfigVariablesIni
 
 const name: string = 'set-admin';
 const signature: string = `${name} [alias-or-address] [new-admin-address]`;
-const description: string = 'change upgradeability admin of a contract instance. Provide the [alias] or [package]/[alias] of the contract to change the ownership of all its instances, or its [address] to change a single one. Note that if you transfer to an incorrect address, you may irreversibly lose control over upgrading your contract.';
+const description: string = `change upgradeability admin of a contract instance, all instances or proxy admin.
+Provide the [alias] or [package]/[alias] of the contract to change the ownership of all its instances,
+or its [address] to change a single one, or none to change all contract instances to a new admin. 
+Note that if you transfer to an incorrect address, you may irreversibly lose control over upgrading your contract.`;
 
 const register: (program: any) => any = (program) => program
   .command(signature, undefined, { noHelp: true })
   .usage('[alias-or-address] [new-admin-address] --network <network> [options]')
   .description(description)
-  .option('-y, --yes', 'accept transferring admin rights (required)')
+  .option('-f, --force', 'bypass a manual check')
   .withNetworkOptions()
   .action(action);
 
 async function action(contractFullNameOrAddress: string, newAdmin: string, options: any): Promise<void | never> {
   const { network, txParams } = await ConfigVariablesInitializer.initNetworkConfiguration(options);
   if (!await hasToMigrateProject(network)) process.exit(0);
 
-  const { yes } = options;
-  if (!yes) {
-    throw Error('This is a potentially irreversible operation: if you specify an incorrect admin address, you may lose the ability to upgrade your contract forever.\nPlease double check all parameters, and run the same command with --yes.');
-  }
-
   let proxyAddress;
   let contractAlias;
   let packageName;
 
+  if (!contractFullNameOrAddress) throw Error('You have to specify at least a new admin address.');
+
+  // we assume if newAdmin is empty it was specified as first argument
+  if (!newAdmin) {
+    newAdmin = contractFullNameOrAddress;
+    contractFullNameOrAddress = '';
+  }
+
+  if (!options.force) {
+    const answers = await inquirer.prompt([
+      {
+        name: 'address',
+        type: 'string',
+        message: 'Please tripe check your address and type the last 4 letters of the new admin address. If you provide a wrong address you will lose control forever.',
+      }
+    ]);
+    if (answers.address !== newAdmin.slice(-4)) {
+      throw Error('Addresses do not much. Pay attention to your address. If you provide wrong address you will lose control forever.');
```

Comment:
```suggestion
      throw Error('Last 4 characters of the admin address do not match');
```
Keep an eye on spelling for `match`!